content/id/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -26,7 +26,7 @@ <h2>Panduan Dasar Kubernetes</h2>
         <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi klaster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan <i>feature</i> Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen klaster sederhana beserta aplikasi dalam kontainer yang kamu miliki.</p>
         <p>Dengan menggunakan mode tutorial interaktif ini, pengguna diharapkan dapat memahami:</p>
         <ul>
-        <li><i>Deploy</i>i> sebuah aplikasi yang sudah dikontainerisasi pada klaster</li>
+        <li><i>Deploy</i> sebuah aplikasi yang sudah dikontainerisasi pada klaster</li>
         <li>Melakukan <i>scale</i> <i>deployment</i></li>
         <li>Meng-update aplikasi yang sudah dikontainerisasi dengan menggunakan versi aplikasi terbaru</li>
```

Comment:
```suggestion
        <li>Meng-<i>update</i> aplikasi yang sudah dikontainerisasi dengan menggunakan versi aplikasi terbaru</li>
```

Reply:
You can use "memperbarui" instead of the English word update

Reply:
```suggestion
        <li>Menperbarui aplikasi yang sudah dikontainerisasi dengan menggunakan versi aplikasi terbaru</li>
```